buildpack/centos7/Dockerfile

```diff
@@ -43,11 +43,12 @@ RUN yum -y install \
 
 
 # Python and tools
-RUN yum -y install python-devel && \
-    wget -qO - https://bootstrap.pypa.io/get-pip.py | python && \
-    pip install --upgrade "pip>=19.0.0,<20.0.0" && \
-    pip install setuptools virtualenv && \
-    rm -rf /root/.cache
+RUN yum -y install python-devel
+RUN yum install -y python3 python3-devel
+RUN wget -qO - https://bootstrap.pypa.io/get-pip.py | python3
+RUN pip install --upgrade "pip==20.3.4"
+RUN pip install setuptools virtualenv
+RUN rm -rf /root/.cache
```

Comment:
Please revert the changes with several commands grouped behind a single `RUN`.
This is within Docker best practices to minimize the amount of layers and reduce the produced Docker image size.

Reply:
Done.